.github/workflows/tox.yml

```
@@ -0,0 +1,39 @@
+---
+name: Python package with Tox
```

Let's make this a bit more explicit :)

```suggestion
name: Test the Python package with Tox
```